### PR TITLE
Option to set the local/client IP

### PR DESCRIPTION
**Pull request by abh**

### CODE REVIEW

File: ntp.go [hunk 1]
```diff
@@ -145,10 +145,11 @@ func (m *msg) getLeapIndicator() LeapIndicator {
 // QueryOptions contains the list of configurable options that may be used with
 // the QueryWithOptions function.
 type QueryOptions struct {
-	Timeout time.Duration // defaults to 5 seconds
-	Version int           // NTP protocol version, defaults to 4
-	Port    int           // NTP Server port for UDPAddr.Port, defaults to 123
-	TTL     int           // IP TTL to use for outgoing UDP packets, defaults to system default
+	Timeout      time.Duration // defaults to 5 seconds
+	Version      int           // NTP protocol version, defaults to 4
+	LocalAddress string        // IP to use for the client address
```
Comment (beevik) [inline ntp.go]: suggested comment:

`// Client local address, nil to choose automatically`

**Comment by beevik** [comment]
Could you also add yourself to the `CONTRIBUTORS` file?

File: ntp.go (hunk 1 above)
Comment (abh) [reply ntp.go]: Done

**Comment by beevik** [comment]
I update the contributors file in a separate change list.  Thanks for the contributions!

**Pull request closed without merging**